source/fundamentals/linq.txt

```diff
@@ -609,6 +608,35 @@ The following is a subset of the documents contained in the returned results:
    { "_id" : ObjectId(...), "name" : "Abaleh", "restaurant_id" : "50009096", "cuisine" : "Mediterranean", "address" : { ... }, "borough" : "Manhattan", "grades" : [...] }
    ...
 
+.. _csharp-linq-convert:
+
+$convert
+~~~~~~~~
+
+You can use the ``Mql.Convert()`` method in a ``Select()`` projection to
```

Comment:
The use of `Mql.Convert` is not limited only to `Select`, even though it could be the most common.